src/python/pants/core/goals/export.py

```diff
@@ -39,6 +39,12 @@ class ExportRequest:
     targets: Targets
 
 
+@union
+@dataclass(frozen=True)
+class ExportToolRequest:
+    pass
```

Comment:
This should live in the Python file. There is no need to add anything generic to this file - it already has everything we need

Reply:
I'm not sure what you mean by "the Python file" here and below

Reply:
Sorry, I am using dictation because of a broken collarbone. So things aren't always as clear 😬

I mean backend/python/goals/export.py. This PR should not require changing core/ at all.

Reply:
Ouch! Sorry to hear that 😬 

Will update as requested 👍